tests/python/unittest/test_gluon_rnn.py

```diff
@@ -490,7 +505,7 @@ def test_layer_fill_shape():
     layer.hybridize()
     check_rnn_layer_forward(layer, mx.nd.ones((3, 2, 7)))
     print(layer)
-    assert layer.i2h_weight[0].shape[1] == 7, layer.i2h_weight[0].shape[1]
+    assert layer.l0_i2h_weight.shape[1] == 7, layer.l0_i2h_weight.shape[1]
```

Comment:
is this API change?

Reply:
This has not been exposed as a documented attribute before and shouldn’t be considered part of API